tests/tiling/README.md

```diff
@@ -0,0 +1,21 @@
+This version of `explosive_source` is not compatible with versions of Firedrake
```

Comment:
Not that it really matters, but most of the URLs are broken when the README is rendered on GitHub: https://github.com/opesci/seigen/tree/merge-tiling/tests/tiling because of the line breaks.